### PR TITLE
fix: #1194 application dropdown should list application in order

### PR DESCRIPTION
**Pull request by J0taFerreira**
- Dropdown for dashboard and application admin now are sorted alphabetically
- Spike if the Primevue component can do sort at the component level(Primeve does not offer an sort for the dropdown component)

### CODE REVIEW

File: frontend/src/store/FamLoginUserState.ts [hunk 1]
```diff
@@ -103,7 +103,9 @@ const getApplicationsUserAdministers = () => {
                 });
             }
         });
-        applicationList.sort((first, second) => first.id - second.id);
+        applicationList.sort((first, second) => {
+            return first.description < second.description ? -1 : 1
+        });
         if (famApp) applicationList.unshift(famApp); // add FAM to the first if FAM Admin.
```
Comment (J0taFerreira) [inline frontend/src/store/FamLoginUserState.ts:109]: this condition was added to put FAM at the top of the list intentionally, should it be removed?

Comment (MCatherine1994) [reply frontend/src/store/FamLoginUserState.ts:109]: Hi Jota, I think we can keep it, cause only us will see that anyway

**Comment by MCatherine1994** [comment]
Hi Jota, I'm checking the order in the application dropdown, but not sure if this looks correct, should we have "SILVA" at the end? maybe we should order them all based on lowercase chars

 Screen Shot 2024-03-19 at 10 43 50 AM

**Comment by MCatherine1994** [comment]
And we need to order the application list on the assign app admin screen as well, thanks!!
Screen Shot 2024-03-19 at 10 44 03 AM

File: frontend/src/store/FamLoginUserState.ts (hunk 1 above)
Comment (J0taFerreira) [reply frontend/src/store/FamLoginUserState.ts:109]: sure, thanks!

**Comment by MCatherine1994** [comment]
> Hi Jota, I'm checking the order in the application dropdown, but not sure if this looks correct, should we have "SILVA" at the end? maybe we should order them all based on lowercase chars
> 
> Screen Shot 2024-03-19 at 10 43 50 AM

Hi Jota, did you check this case?

**Comment by MCatherine1994** [comment]
Hi Jota, do you mind check this on your local again? cause I'm not seeing the application list is in order when grant app admin, thanks!!

**Comment by MCatherine1994** [comment]
Hi Jota, let's check the sonar cloud, it has 4 new issues, thanks!!

**Comment by J0taFerreira** [comment]
> Hi Jota, let's check the sonar cloud, it has 4 new issues, thanks!!

Hello! Just did, there's only the test code coverage. Thanks!

**Review by MCatherine1994 — APPROVED** [review]

**Comment by MCatherine1994** [comment]
thanks Jota, looks good!

**Pull request merged**